python/Dockerfile

```diff
@@ -1,8 +1,11 @@
-FROM python:3.8 
+FROM python:3.8.13-slim-buster 
```

Comment:
I am not sure if you checked, but this makes the resulting image considerably smaller.
Nicely done!